docs/usage/getting-started/use-cases.md

```diff
@@ -177,9 +177,9 @@ You can set the time ranges during which Renovate creates updates in the `schedu
 
 You can use Renovate's "Dependency Dashboard" on platforms which support dynamic Markdown checkboxes:
 
+- Gitea
```

Comment:
The Renovate docs homepage lists the platforms Renovate supports:

https://github.com/renovatebot/renovatebot.github.io/blob/574afae0aa5f4e5990254acb6e2eaa287aea00ae/src/index.md?plain=1#L22-L32

Do we need to add Forgejo to the list of supported platforms with a separate PR on the docs repository?

Also do we want to style things like `Gitea/Forgejo` or have separate entries for each platform?

Reply:
- #19939